packages/react-components/react-tree/src/components/TreeItemPersonaLayout/TreeItemPersonaLayout.types.ts

```diff
@@ -24,10 +21,10 @@ export type TreeItemPersonaLayoutSlots = {
    * Secondary text that describes or complements the main text
    */
   description?: Slot<'div'>;
-  /**
-   * A layout wrapper for the main and description slots
-   */
-  content: NonNullable<Slot<'div'>>;
+};
+
+export type TreeItemPersonaLayoutInternalSlots = TreeItemPersonaLayoutSlots & {
```

Comment:
This is required to ensure we're properly removing the `visible` property, that is available by the external props, from the actual internal slot (that should not have that property propagated to the DOM)